pkgs/by-name/vo/volanta/package.nix

```
@@ -0,0 +1,39 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+  makeWrapper,
+}:
+let
+  pname = "volanta";
+  version = "1.10.10";
+  src = fetchurl {
+    url = "https://cdn.volanta.app/software/volanta-app/${version}-a7ebf1c7/volanta-${version}.AppImage";
+    hash = "sha256-pdMPC3flJzguRYmS+xKiAXWQ4BKsD3N48S27djXDtuo=";
+  };
+  appImageContents = appimageTools.extract { inherit pname version src; };
+in
+appimageTools.wrapType2 rec {
+  inherit pname version src;
+
+  nativeBuildInputs = [ makeWrapper ];
+  # Installs the .desktop file to the appropriate location so that it can be found by launchers, etc.
+  # Note: Volanta needs the env variable APPIMAGE=true to be set in order to work at all.
+  extraInstallCommands = ''
+    install -m 444 -D ${appImageContents}/volanta.desktop $out/share/applications/volanta.desktop
+    install -m 444 -D ${appImageContents}/volanta.png \
+      $out/share/icons/hicolor/1024x1024/apps/volanta.png
+    substituteInPlace $out/share/applications/volanta.desktop \
+    --replace-fail 'Exec=AppRun' 'Exec=env APPIMAGE=true ${meta.mainProgram}'
```

```suggestion
      --replace-fail 'Exec=AppRun' 'Exec=env APPIMAGE=true volanta'
```
changing meta.mainProgram will not rename that and we should not make it rename the file